pepper-apis/src/main/java/org/broadinstitute/ddp/housekeeping/schedule/CheckKitsJob.java

```diff
@@ -54,25 +66,15 @@ public static void register(Scheduler scheduler, Config cfg) throws SchedulerExc
         scheduler.addJob(job, true);
         LOG.info("Added job {} to scheduler", getKey());
 
-        if (!cfg.getBoolean(ConfigFile.Kits.CHECK_ENABLED)) {
-            LOG.warn("Job {} is disabled, no trigger added", getKey());
-            return;
-        }
-
         int intervalSecs = cfg.getInt(ConfigFile.Kits.INTERVAL_SECS);
         Trigger trigger = TriggerBuilder.newTrigger()
                 .withIdentity(Keys.Kits.CheckTrigger)
                 .forJob(getKey())
                 .withSchedule(SimpleScheduleBuilder.repeatSecondlyForever(intervalSecs))
-                .startNow()
+                .startAt(new Date(Instant.now().toEpochMilli() + INITIAL_TRIGGER_DELAY_SECS * 1000))
                 .build();
         scheduler.scheduleJob(trigger);
         LOG.info("Added trigger {} for job {} with delay of {} seconds", trigger.getKey(), getKey(), intervalSecs);
-
-        kitCheckService = new KitCheckService(cfg.getInt(ConfigFile.Kits.BATCH_SIZE));
```

Comment:
We were getting a NPE because this object was initialized AFTER we added a trigger. Once the trigger was added, it ran the job and hit a null object.